opencanary/logger.py

```diff
@@ -99,7 +99,8 @@ class LoggerBase(object):
 
     def sanitizeLog(self, logdata):
         logdata['node_id'] = self.node_id
-        logdata['local_time'] = datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S.%f")
+        logdata['UTC_time'] = datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S.%f")
```

Comment:
so lets stick with the lowercase convention please, `logdata['utc_time']`

Reply:
done